qcodes/instrument_drivers/Galil/dmc_41x3.py

```diff
@@ -0,0 +1,964 @@
+            docstring="can be used to set software forward limit for the motor."
+                      " motor motion will stop beyond this limit automatically."
+                      " default value is 2147483647. this value effectively "
+                      "disables the forward limit.",
+        )
+
+    def _set_reverse_sw_limit(self, val: int) -> None:
+        """
+        Sets reverse software limit
+        """
+        self.write(f"BL{self._axis}={val}")
+
+    def _set_forward_sw_limit(self, val: int) -> None:
+        """
+        Sets forward software limit
+        """
+        self.write(f"FL{self._axis}={val}")
+
+    def _get_off_when_error_occurs(self) -> int:
+        """
+        Gets the status if motor is automatically set to turn off when error
+        occurs.
+        """
+        val = self.ask(f"MG _OE{self._axis}")
+
+        return int(val[0])
+
+    def _set_off_when_error_occurs(self, val: int) -> None:
+        """
+        sets the motor to turn off automatically when the error occurs
+        """
+        self.write(f"OE{self._axis}={val}")
+
+    def _set_deceleration(self, val: str) -> None:
+        """
+        set deceleration for the motor's motion
+        """
+        self.write(f"DC{self._axis}={val}")
+
+    def _set_acceleration(self, val: str) -> None:
+        """
+        set acceleration for the motor's motion
+        """
+        self.write(f"AC{self._axis}={val}")
+
+    def _set_speed(self, val: str) -> None:
+        """
+        sets speed for motor's motion
+        """
+        self.write(f"SP{self._axis}={val}")
+
+    def _set_relative_position(self, val: str) -> None:
+        """
+        sets relative position
+        """
+        self.write(f"PR{self._axis}={val}")
+
+    def off(self) -> None:
+        """
+        turns motor off
+        """
+        self.write(f"MO {self._axis}")
+
+    def on_off_status(self) -> str:
+        """
+        tells motor on off status
+        """
+        val = self.ask(f"MG _MO{self._axis}")
+        if val[0] == "1":
+            return "off"
+        else:
+            return "on"
+
+    def servo_here(self) -> None:
+        """
+        servo at the motor
+        """
+        self.write(f"SH {self._axis}")
+
+    def begin(self) -> None:
+        """
+        begins motion of the motor and waits until motor stops moving
+        """
+        self.write(f"BG {self._axis}")
+
+    def wait_till_motor_motion_complete(self) -> None:
+        """
+        this method on call waits for motion to complete on the motor
+        """
+        try:
+            while int(float(self.ask(f"MG _BG{self._axis}"))):
+                pass
+        except KeyboardInterrupt:
+            self.root_instrument.abort()
+            self.off()
+
+    def error_magnitude(self) -> float:
+        """
+        gives the magnitude of error, in drive step counts, for axes
+
+        a step count is directly proportional to the micro-stepping
```

Comment:
do we need an additional conveience method that return this value in more useful units? perhaps, in quadrature counts? or is that not useful/relevant/possible?